kgbrules/status_rules.py

```diff
@@ -0,0 +1,90 @@
+"""Rules applied to the first line of the commit."""
+
+import re
+
+from .config import MAX_STATUS_LENGTH
+from .config import TYPES
+
+_STATUS_PATTERN = re.compile(
+    r"^([^()\s]*)(\s*)\(([^()\s]*)\)(\s*):(\s*)(.*\w)(\s*)$")
+
+_TYPE_STRING = ", ".join(sorted(TYPES))
+
+
+def check_status_length(status_line):
+    """Check status length.
+
+    >>> check_status_length("this is short enough")
+
+    >>> check_status_length("TOO_LONG" * 10)
+    'Limit status to 50 characters'
+    """
+    if len(status_line) > MAX_STATUS_LENGTH:
+        return "Limit status to {0} characters".format(MAX_STATUS_LENGTH)
+
+
+def check_status_formatting(status_line):
```

Comment:
After consideration. 
It doesn't seem to make sense to split this function into simpler components.
I believe it is understandable and will leave it as is.